client/allocrunner/testing.go

```diff
@@ -99,7 +100,7 @@ func testAllocRunnerConfig(t *testing.T, alloc *structs.Allocation) (*Config, fu
 	return conf, cleanup
 }
 
-func TestAllocRunnerFromAlloc(t *testing.T, alloc *structs.Allocation) (*allocRunner, func()) {
+func TestAllocRunnerFromAlloc(t *testing.T, alloc *structs.Allocation) (interfaces.AllocRunner, func()) {
```

Comment:
Do we need to update this test helper? It seems like returning the "real" implementation is the whole point.